server/libs/index.js

```diff
@@ -4,6 +4,7 @@ module.exports = function initNodeLibraries(conf) {
   require('./express')(libs, conf);
   require('./sequelize')(libs, conf);
   require('./sequelize-session')(libs, conf);
+  require('./statistic')(libs, conf);
```

Comment:
@NoSkiiilL @phong-steph 
The `statistics` module is not an external library like `express` or `sequelize`. It is an internal functionality. What you included from this `require` should be moved to the `managers` section.